src/faq/_faqAccordion.scss

```diff
@@ -0,0 +1,13 @@
+.FaqAccordion {
+  margin-top: $spacing-l;
+  min-width: 60%;
+  border-bottom: none !important;
+}
+
+.FaqItem {
+  margin-bottom: $spacing-m;
```

Comment:
Consider also adding a whitespace rule to allow newlines added in django admin to also show up here.